### DISCUSSION TITLE
Why can't be used in cloud environment?

### DISCUSSION THREAD

**Discussion by NikoKS** [onedr0p/cluster-template#989] [Q&A] [answered]
Hello, the README says that

```
this is NOT designed for cloud environments.
```

Any reason why that is? I want to try to learn k3s and flux using this repo, but don't have a homelab setup. I was wondering if I can just rent a 2core 4gb ram VPS to try it out.

**Comment by onedr0p** [comment, marked as answer]
Overall this _can_ be used with a cloud provider with some modifications but I don't offer any support for that as I try to keep it opinionated for home use. Adding cloud support would be a larger support burden than I can handle.
